src/common/util/dictionary.js

```diff
@@ -1,9 +1,9 @@
 const https = require("https");
-const fs = require('fs');
+const fs = require("fs");
 
 // TODO: make this a config thing and simplify paths
-const CACHE_DIR  = './local';
-const DICT_FILE_PATH = CACHE_DIR + '/dictionary.txt';
+const CACHE_DIR  = "./local";
+const DICT_FILE_PATH = CACHE_DIR + "/dictionary.txt";
 
```

Comment:
we need to make our own dictionary. every asset we use needs to be created by us or we need permission to use it. we can't depend on this random resource we found online to be around forever